devtools-ci-index.yaml

```diff
@@ -556,6 +556,10 @@
                 - openshift-connector
     triggers:
         - github
+        - github-pull-request:
```

Comment:
Please open a second PR for that. The verification job is already enough to review/test for that PR. Triggers for the existing che job and the introduction of a new verification job are different things and should be delivered in 2 distincts commits for clarity and consistency.

When you open the new PR keep in mind that:
- the github-pull-request trigger should be restricted to a list of user 
- the trigger-phrase should be bracket-enclosed (e.g. `[test]`)